### ISSUE TITLE
[Issue]: Infuse Blinking and Genre Bug

### ISSUE THREAD

**Issue by dannymichel** [jellyfin/jellyfin#7773]
### Please describe your bug

Not sure if the genre bug is a jellyfin thing or infuse thing, but in this video you’ll see one of the tv show posters constantly blinking and the fact that I have more than 13 genres of tv shows on Jellyfin but infuse only shows 13. Once I go back to the app you’ll notice a very long refresh. I stop recording after a significant enough time.
https://youtu.be/YxLGMZfzymE

### Jellyfin Version

10.8.0

### Environment

```markdown
- OS: Linux
- Virtualization: Docker
- Clients: iOS
- Browser: n/a
- FFmpeg Version: n/a
- Playback Method: n/a
- Hardware Acceleration: No
- Plugins: Trakt, Infuse, Opensubtitles
- Reverse Proxy: Nginx
- Base URL: n/a
- Networking: n/a
- Storage: rclong/gcloud/mergerfs
```

### Please attach any screenshots here

https://youtu.be/YxLGMZfzymE

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dannymichel** [comment]
> Neither of these are Jellyfin server bugs.

The genre thing is. Tv show genres are limited on Jellyfin and not on plex/google drive direct/emby for infuse. Genres are limited to 13 for tv shows.

**Comment by crobibero** [comment]
Opening the genres tab on the tv shows view lists 20+ genres for me in the web client.

**Comment by dannymichel** [comment]
> Opening the genres tab on the tv shows view lists 20+ genres for me in the web client.

Yes definitely as you can see in the video it shows all the genres in the web client but doesn't in infuse.
The reason it's a Jellyfin issue is because plex and everything else shows all the tv show genres on infuse

**Comment by crobibero** [comment]
It is not a Jellyfin issue. This is an Infuse issue.
The web client showing all of the genres proves this.
